lib/google.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+create_image_from_snapshot(){
+    IMAGE=$1
+    FAMILY=$2
+    DESCRIPTION=$3
+    SNAPSHOT=$4
+    gcloud compute images create "$IMAGE" --family "$FAMILY" --description "$DESCRIPTION" --source-snapshot "$SNAPSHOT"
+}
+
+create_image_from_bucket(){
+    IMAGE=$1
+    FAMILY=$2
+    DESCRIPTION=$3
+    IMAGE_FILE=$4
+    gcloud compute images create "$IMAGE" --family "$FAMILY" --description "$DESCRIPTION" --source-uri "gs://$BUCKET_NAME/$IMAGE_FILE"
+}
+
+delete_image(){
+    IMAGE=$1
+    image_name=$(gcloud compute images list --project $GCE_PROJECT --filter "family = $FAMILY AND name = $IMAGE" --format json | jq -r '.[]|.name')
```

Comment:
Where does FAMILY and GCE_PROJECT come from?

Reply:
GCE_PROJECT is the name of our project in gce, and family is a name which we use to select images, for example, we could have ubuntu-16-v1 and ubuntu-16-v2, both with family ubuntu-16, so when we declare the system we can use ubuntu-16 and then spread will automatically take the last image with this family.